modules/api-svc/src/FSPIOPEventHandler/handlers/party-info-requested.js

```diff
@@ -56,7 +56,11 @@ module.exports.handlePartyInfoRequestedDmEvt = async (
             bulkId: event.getKey(),
             content: {
                 transferId: event.getTransferId(),
-                partyResult: response,
+                partyResult: {
+                    party: response.party?.body,
```

Comment:
I don't think there is a `body` in the response:
https://github.com/mojaloop/sdk-scheme-adapter/blob/16ccfe52f101d38daa397cf3312f33462bf8176c/src/OutboundServer/handlers.js#L467-L475

Reply:
Hi @kirgene , there are body and headers in party. Yest that's weird, but its the response from model currently. So I made this transformation as workaround

Reply:
Thats a carry over from merging the mojaloop-connector into the sdk-scheme-adapter. The `headers` were needed in the synchronous responses so now the schema is typically `objectResponse: {body:body, headers:headers}`.

Reply:
Confusing I know v - v but that was the way to keep everything standardized.